lib/teracy-dev-essential/config/host_manager.rb

```diff
@@ -31,9 +35,48 @@ def configure_node(settings, config)
 
       private
 
+      def configure_ip_display(config, settings)
+        extension_lookup_path = TeracyDev::Util.extension_lookup_path(settings, 'teracy-dev-essential')
+
+        networks = get_all_configured_networks(settings)
+
+        # default VM network (127.0.0.1) is at 0 index
+        # we use the latest defined network IP
+        host_ip_index = networks.length
```

Comment:
I'm not sure this is reliable because the network can be created dynamically without using the networks config (using vagrant config or running shell commands within the VM), we must make sure the feature should work with those cases.